swagger/v2/get_people_responses.yaml

```diff
@@ -11,17 +11,15 @@
       type: array
       items:
         anyOf:
-        - $ref: gender.yaml#/Gender
-        - $ref: ethnicity.yaml#/Ethnicity
-        - $ref: profile.yaml#/Profile
```

Comment:
I guess we'll implement this on the move

Reply:
if you refer to the profile, I'm going to add it back, since it is a supported relationship